cli/config/compose/profiles/ingest-manager/docker-compose.yml

```diff
@@ -29,13 +29,13 @@ services:
       test: "curl -f http://localhost:5601/login | grep kbn-injected-metadata 2>&1 >/dev/null"
       retries: 600
       interval: 1s
-    image: "docker.elastic.co/kibana/kibana:8.0.0-SNAPSHOT"
+    image: "docker.elastic.co/kibana/kibana:${stackVersion:-8.0.0-SNAPSHOT}"
     ports:
       - "5601:5601"
     volumes:
       - ${kibanaConfigPath}:/usr/share/kibana/config/kibana.yml
   package-registry:
-    image: docker.elastic.co/package-registry/package-registry:master
+    image: docker.elastic.co/package-registry/package-registry:41c150c8020efc53ab16e3bba774c62a419b51ea
```

Comment:
can we get a comment as to how folks would know how and when to update this... I think some details will be here: https://github.com/elastic/package-storage/issues/86

Reply:
We replaced master branch because there was a combination of errors:
- Kibana was unable to start
- the enrollment process of agents did not work because of different agent versions compared on Kibana

I noticed the team changed the version here: https://github.com/elastic/integrations/commit/25a4bd4e160afd3ed763c88745d265cde30941e3, and I shadowed it.